src/types/actions/workflow-step-edit.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * A Slack workflow step action wrapped in the standard metadata.
+ *
+ * This describes the entire JSON-encoded body of a request from Slack workflow step actions.
+ */
+export interface WorkflowStepEdit {
+  type: 'workflow_step_edit';
+  callback_id: string;
+  trigger_id: string;
+  user: {
+    id: string;
+    username: string;
```

Comment:
This was changed from `name` to `username` based on the payloads

Reply:
Thanks for catching and fixes all of those 😄